config.py

```diff
@@ -38,7 +38,6 @@ class Config(object):
     DM_LOG_LEVEL = 'DEBUG'
     DM_PLAIN_TEXT_LOGS = False
     DM_APP_NAME = 'admin-frontend'
-    DM_DOWNSTREAM_REQUEST_ID_HEADER = 'X-Amz-Cf-Id'
```

Comment:
This isn't mentioned in your commit message, but I assume it's an intentional re-removal of the obsolete ting, or something like that.